src/main/java/com/github/tomakehurst/wiremock/store/InMemoryObjectStore.java

```diff
@@ -27,6 +32,7 @@ public class InMemoryObjectStore implements ObjectStore {
   private final ConcurrentHashMap<String, Object> cache;
   private final Queue<String> keyUseOrder = new ConcurrentLinkedQueue<>();
   private final int maxItems;
+  private final List<StoreEventHandler<String, Object>> listeners = new ArrayList<>();
```

Comment:
Any reason we can't just use `Consumer<StoreEvent>` as the listener type?

Reply:
yep sounds good to me

Reply:
done